onnxruntime/contrib_ops/cuda/custom_layer_norm.cuh

```diff
@@ -0,0 +1,49 @@
+/**
+* Copyright (c) 2016-present, Facebook, Inc.
```

Comment:
Same here... Why FB and Nvidia header here?

Reply:
removed FB and nvidia headers.